gnovm/cmd/gno/util.go

```diff
@@ -86,9 +86,14 @@ func gnoPackagesFromArgs(args []string) ([]string, error) {
 				}
 				visited[parentDir] = true
 
-				// cannot use path.Join or filepath.Join, because we need
-				// to ensure that ./ is the prefix to pass to go build.
-				pkg := "./" + parentDir
+				pkg := parentDir
+				if !filepath.IsAbs(parentDir) {
+					// cannot use path.Join or filepath.Join, because we need
+					// to ensure that ./ is the prefix to pass to go build.
+					// if not absolute.
+					pkg = "./" + parentDir
+				}
```

Comment:
@harry-hov I've just noticed there's a bug in `gno test` that avoids the usage of absolute path in arguments.

For instance, this doesn't work, although the path passed in argument is correct :
```
$ gno test /home/tom/src/gno/examples/gno.land/r/demo/tests
?       .//home/tom/src/gno/examples/gno.land/r/demo/tests 	[no test files]
```

Your change seems to fix that, can you confirm please ?

Reply:
Yep. I fixed it in this PR. 

Thanks for reminding me. Will include it in the PR description.

Reply:
Great thank you!